Frontend/src/pages/about/about.tsx

```diff
@@ -30,34 +30,34 @@ const AboutUs: React.FC = () => {
         </p>
         <p className={styles.textBlock}>
           Proto TI s vizí vytvoření funkční a aktualizované databáze spojila síly s IT profesionály na otevřené zdroje z
-          Hlídače státu a komunitou špičkových vývojářů, designérů a produkťáků z česko.digital. Díky této jedinečné
-          spolupráci jsme reinkarnovali projekt, který by měl být místem, kde najdete ověřené informace o Vámi volených
+          Hlídače státu a komunitou špičkových vývojářů, designérů a produkťáků z Česko.Digital. Díky této jedinečné
+          spolupráci jsme reinkarnovali projekt, který by měl být místem, kde najdete ověřené informace o vámi volených
           zástupcích.
         </p>
         <h1 className={styles.title}>Jak číst jednotlivé datové zdroje?</h1>
         <h2 className={styles.subTitle}>Výroky</h2>
         <p className={styles.textBlock}>
           Výroky naznačují, jak pravdivě či nepravdivě se vyjadřuje politik/politička, kterého hledáte. Jedná se o data
           organizace Demagog.cz, která ověřuje výroky politiků (tzv. factchecking) a snaží se tak kultivovat veřejnou
-          debatu. Nehodnotí se politická prohlášení, hodnotící soudy a predikce dopadů nějakých opatření (např.:
-          Opatření vlády XY je dobré, neboť způsobí růst zaměstnanosti.), jednoduše proto, že to není možné ověřit:
-          budoucnost neznáme nikdo a u hodnotových soudů nelze nalézt jedinou „správnou“ pozici. Více o metodologii dat
-          se dočtete na stránkách Demagog.cz.
+          debatu. Nehodnotí se politická prohlášení, hodnotící soudy a predikce dopadů nějakých opatření („opatření vlády
+          XY je dobré, neboť způsobí růst zaměstnanosti), jednoduše proto, že je není možné ověřit – budoucnost neznáme
+          nikdo a u hodnotových soudů nelze nalézt jedinou „správnou“ pozici. Více o metodice dat se dočtete na stránkách
+          Demagog.cz.
```

Comment:
Nebylo by dobré dát sem rovnou <a> na Demagog?

Reply:
Bylo. Napadlo mě to, bál jsem se dělat větší změny naslepo (neznám ten formát a nezkouším to naživo), ale udělám.